test/csuite/wt7989_compact_checkpoint/main.c

```diff
@@ -91,21 +94,21 @@ main(int argc, char *argv[])
     run_test_clean(true, false, opts->preserve, opts->home, "SR", opts->uri);
 
     /*
-     * Now, run test where compact and checkpoint threads are synchronized using condition variable.
-     * Row store case.
+     * Now, run test where compact and checkpoint threads are synchronized using global thread
+     * counter. Row store case.
      */
     run_test_clean(false, false, opts->preserve, opts->home, "NR", opts->uri);
 
     /*
      * Next, run test with WT_TIMING_STRESS_CHECKPOINT_SLOW. Column store case.
      */
-    run_test_clean(true, true, opts->preserve, opts->home, "SC", opts->uri);
+    // run_test_clean(true, true, opts->preserve, opts->home, "SC", opts->uri);
```

Comment:
I think you didn't mean to comment out this test and the one below.

Reply:
Ah, my mistake, thanks for that.